### PR TITLE
Adding original Order Id to details list, renaming Order Id to Purchase Id everywhere

### PR DESCRIPTION
**Pull request by irinatarn**
image

### CODE REVIEW

File: frontend/src/components/items-details-list/items-details-list.tsx [hunk 1]
```diff
@@ -83,7 +83,7 @@ export class ItemsDetailsList extends Component<ItemsDetailsListProps, never> {
         },
         {
             key: ITEM_LIST_COLUMN_KEYS.ORDER_ID,
-            name: 'Order ID',
```
Comment (satyamamit) [inline frontend/src/components/items-details-list/items-details-list.tsx:86]: There are few more files which i see has order ID

image

File: frontend/src/screens/review-console/item-details/parts/link-analysis/items-list/items-list.tsx [hunk 2]
```diff
@@ -101,6 +101,37 @@ export class ItemsList extends Component<ItemsListComponentProps, never> {
...
+            maxWidth: 260,
+            isCollapsible: true,
+            isRowHeader: true,
+            isPadded: true,
+            className: `${CN}__cell`,
+            columnActionsMode: ColumnActionsMode.disabled,
+            onRender: (item: LinkAnalysisItem) => {
+                const itemId = isMrItem(item)
+                    ? item.item.purchase.originalOrderId || ''
+                    : item.purchaseId;
```
Comment (satyamamit) [inline frontend/src/screens/review-console/item-details/parts/link-analysis/items-list/items-list.tsx:118]: why you have this condistion here. is originalOrderId not coming as customer attributes if yes, then DFP data should also have this.

File: frontend/src/screens/review-console/item-details/parts/link-analysis/items-list/items-list.tsx [hunk 3]
```diff
@@ -101,6 +101,37 @@ export class ItemsList extends Component<ItemsListComponentProps, never> {
...
+
+                return (
+                    <div className={`${CN}__order-id-cell`}>
```
Comment (satyamamit) [inline frontend/src/screens/review-console/item-details/parts/link-analysis/items-list/items-list.tsx:121]: is this also showing only on hover ? can you please share screenshot here.

**Review by satyamamit — CHANGES_REQUESTED** [review]

File: frontend/src/screens/review-console/item-details/parts/link-analysis/items-list/items-list.tsx (hunk 2 above)
Comment (irinatarn) [reply frontend/src/screens/review-console/item-details/parts/link-analysis/items-list/items-list.tsx:118]: YOu are right

File: frontend/src/screens/review-console/item-details/parts/link-analysis/items-list/items-list.tsx (hunk 3 above)
Comment (irinatarn) [reply frontend/src/screens/review-console/item-details/parts/link-analysis/items-list/items-list.tsx:121]: copy

File: frontend/src/components/items-details-list/items-details-list.tsx (hunk 1 above)
Comment (irinatarn) [reply frontend/src/components/items-details-list/items-details-list.tsx:86]: Corrected.

**Comment by haixiehx** [comment]
Is it possible to update the PR title to be more informative? in the future it's easier to check the commit histories.

File: frontend/src/screens/review-console/item-details/parts/link-analysis/items-list/items-list.tsx [hunk 4]
```diff
@@ -101,6 +101,35 @@ export class ItemsList extends Component<ItemsListComponentProps, never> {
                 );
             }
         },
+        {
+            key: ITEM_LIST_COLUMN_KEYS.ORIGINAL_ORDER_ID,
+            name: 'Original Order ID',
```
Comment (haixiehx) [inline frontend/src/screens/review-console/item-details/parts/link-analysis/items-list/items-list.tsx:106]: nit: should "Order" be lower case? "Original order ID", to be consistent with other columns like "Fraud score", "Import date" etc.

File: frontend/src/constants/search.ts [hunk 5]
```diff
@@ -35,7 +35,8 @@ export const ITEM_LIST_COLUMN_KEYS = {
     STATUS: 'STATUS',
     ANALYST: ITEM_SORTING_FIELD.LABEL_AUTHOR_ID,
     TAGS: 'TAGS',
-    NOTES: 'NOTES'
+    NOTES: 'NOTES',
+    ORIGINAL_ORDER_ID:'ORIGINAL_ORDER_ID'
```
Comment (haixiehx) [inline frontend/src/constants/search.ts:39]: nit: a space between "ORIGINAL_ORDER_ID:" and " 'ORIGINAL_ORDER_ID'"
eslint might complain if you have it ON.

**Review by haixiehx — APPROVED** [review]

File: frontend/src/screens/review-console/item-details/parts/link-analysis/items-list/items-list.tsx (hunk 4 above)
Comment (irinatarn) [reply frontend/src/screens/review-console/item-details/parts/link-analysis/items-list/items-list.tsx]: Fixed.

File: frontend/src/constants/search.ts (hunk 5 above)
Comment (irinatarn) [reply frontend/src/constants/search.ts]: Fixed

**Review by haixiehx — APPROVED** [review]

**Review by satyamamit — APPROVED** [review]

**Pull request merged**